BoostRunClub/Coordinators/BasicCoordinator.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  BasicCoordinator.swift
+//  BoostRunClub
+//
+//  Created by 김신우 on 2020/12/20.
+//
+
+import Combine
+import UIKit
+
+class BasicCoordinator<ResultType>: Coordinator {
+    typealias CoordinationResult = ResultType
+
+    let identifier = UUID()
+    var navigationController: UINavigationController
```

Comment:
```suggestion
    let navigationController: UINavigationController
```

BasicCoordinator init 시에 navigationController를 할당하고 같은 객체를 계속 사용하게 되는데요.
더 이상 변경이 없음을 명확하게 알려주기 위해서 let 을 쓰는게 좋을 것 같아요.